docs/BEMAN_STANDARD.md

```diff
@@ -422,3 +452,21 @@ copyright notice following the SPDX license identifier.
 
 **[CPP.NAMESPACE]** RECOMMENDATION: Headers in `include/beman/<short_name>/` should export
 entities in the `beman::<short_name>` namespace.
+
+**[CPP.NO_FLAG_FORKING]** REQUIREMENT: C++ preprocessing must produce identical
+output regardless of compiler flags.
+
+Therefore, feature test macros such as `__cpp_explicit_this_parameter` should
+not be used directly. Instead use the following approach for feature-dependent
+code generation:
+
+1. Check for availability at CMake time using, for example,
+   `check_cxx_source_compiles`
+2. Create a CMake `option` (e.g. `BEMAN_&lt;short_name&gt;_USE_DEDUCING_THIS`)
+   with a default value based on detected support.
+3. Generate a `config.hpp` with a `#define` macro set to the selected option.
+4. Use this macro in place of the feature test macro.
+
+See
```

Comment:
Please use the convention `Use the following style: ...` (check the rest of the doc)

Reply:
Reminder